pages/Getting Started/Installation.md

```diff
@@ -370,10 +370,11 @@ installed._
 ### Meson
 
 ```plain
-meson subprojects update --reset
+git clone --recursive https://github.com/hyprwm/Hyprland
+cd Hyprland
 meson setup build
-ninja -C build
-ninja -C build install --tags runtime,man
+meson compile -C build
+meson install -C build --skip-subprojects
```

Comment:
Does this not mean udis86 will not be found?

Reply:
udis86 is not part of Hyprland plugin API unlike wlroots was before aquamarine. `--skip-subprojects` is still necessary to avoid installing hyprland-protocols (plugins use preprocessed headers instead). Hyprland itself doesn't support building against system-wide copy of udis86 ([Alpine patches this](https://gitlab.alpinelinux.org/alpine/aports/-/blob/master/testing/hyprland/use-system-installed-udis86.patch)), and the bundled version uses CMake to build the submodule.

Reply:
That's fine by me. However, I've made progress towards allowing Hyprland to build with a system-wide copy of udis86 (provided it's canihavesomecoffee's fork, which includes a .pc file). See this PR for more info about it, maybe leave your thoughts as well https://github.com/hyprwm/Hyprland/pull/7885.